packages/scandipwa/src/component/CartCoupon/CartCoupon.component.js

```diff
@@ -28,7 +28,9 @@ export class CartCoupon extends PureComponent {
         handleApplyCouponToCart: PropTypes.func.isRequired,
         handleRemoveCouponFromCart: PropTypes.func.isRequired,
         mix: MixType.isRequired,
-        title: PropTypes.string.isRequired
+        title: PropTypes.string.isRequired,
+        isBadCoupon: PropTypes.bool.isRequired,
+        resetIsBadCoupon: PropTypes.func.isRequired
```

Comment:
Why coupon is bad? :D Lets make it more semantically correct. Something like 'isIncorrectCoupon'